static/files/latest-multipass-releases.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Source of truth in [original repo](https://github.com/canonical/multipass.run/blob/main/static/latest-release.json)